src/status_im2/contexts/quo_preview/list_items/community_list.cljs

```diff
@@ -0,0 +1,132 @@
+(ns status-im2.contexts.quo-preview.list-items.community-list
+  (:require [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
```

Comment:
FYI: Aliasing as `quo.theme` to avoid having the namespace be shadowed by `theme` bindings.